cibyl/exceptions/__init__.py

```diff
@@ -17,15 +17,21 @@
 
 
 class CibylException(Exception):
+    def __init__(self, message):
```

Comment:
I have two questions about this:
1. Do we really need the `__init__`?
2. Why not `super().__init__(message)`?

Reply:
1.- Not entirely, but Exception's default constructor is: (*args, **kwargs) and I got a warning from Pylint for it.
2.- Same reason, Pylint warned that *args did not match a str.

Reply:
ok, makes sense. I would pass it as kwarg, still. IMO it's clearer